distribution/lib/Standard/Base/0.0.0-dev/src/Data/Statistics.enso

```diff
@@ -145,9 +145,12 @@ type Statistic
         Statistic.Maximum -> Java_Statistic.Maximum
         Statistic.Sum -> Java_Statistic.Sum
         Statistic.Mean -> Java_Statistic.Mean
-        Statistic.Variance _ -> Java_Statistic.Variance
-        Statistic.Standard_Deviation _ -> Java_Statistic.StandardDeviation
-        Statistic.Skew _ -> Java_Statistic.Skew
+        Statistic.Variance True -> Java_Statistic.VarianceTrue
+        Statistic.Variance False -> Java_Statistic.VarianceFalse
+        Statistic.Standard_Deviation True -> Java_Statistic.StandardDeviationTrue
+        Statistic.Standard_Deviation False -> Java_Statistic.StandardDeviationFalse
+        Statistic.Skew True -> Java_Statistic.SkewTrue
+        Statistic.Skew False -> Java_Statistic.SkewFalse
```

Comment:
Could we call the Java one `VarianceSample` and `VariancePopulation`.

Reply:
Done